qutip/core/superop_reps.py

```diff
@@ -500,19 +512,22 @@ def to_kraus(q_oper, tol=1e-9):
 
 def to_stinespring(q_oper, threshold=1e-10):
     r"""
-    Converts a Qobj representing a quantum map $\Lambda$ to a pair of partial
-    isometries $A$ and $B$ such that $\Lambda(X) = \Tr_2(A X B^\dagger)$ for
-    all inputs $X$, where the partial trace is taken over a a new index on the
-    output dimensions of $A$ and $B$.
+    Converts a Qobj representing a quantum map :math:`\Lambda` to a pair of
+    partial isometries ``A`` and ``B`` such that
+    :math:`\Lambda(X) = \Tr_2(A X B^\dagger)` for all inputs ``X``, where the
+    partial trace is taken over a a new index on the output dimensions of
+    ``A`` and ``B``.
 
-    For completely positive inputs, $A$ will always equal $B$ up to precision
-    errors.
+    For completely positive inputs, ``A`` will always equal ``B`` up to
+    precision errors.
 
     Parameters
     ----------
     q_oper : Qobj
         Superoperator to be converted to a Stinespring pair.
 
+    threshold : float, default: 1e-10
+
```

Comment:
It would be good to write a line saying what the threshold does, but I don't know what to write.